### ISSUE TITLE
Combine approx arguments

### ISSUE THREAD

**Issue by sashahafner** [AU-BCE-EE/ABM#23]
```
  approx_method_temp = 'linear', # NTS: combine these in 1 vector?
  approx_method_pH = 'linear',
  approx_method_vent_air = 'linear',
```

Instead:

```
approx_method = c(temp = 'linear', pH = 'constant'...
```

Remove vent_air, not used

**Comment by fdalby** [comment]
removed vent_air and vectorized approx_method #3106e89007d72ec9a48e8a17ee9f835b8aa6312f

**Comment by fdalby** [comment]
@sashahafner another issue related to this. For abm_variable slurry_mass is be default using some kind of 'constant' (hence the warning "removals changed to instant", when you use variable slurry mass. 

I think we made it so, because slurry is removed at once in big portions. But it is a little problematic when we receive measured slurry heights from storage and we actually don't know when slurry was removed between two height measurements. In that case the best guess is in the middle of the two measurements. So i suggest that we apply an argument for slurry interpolation? Could be added to approx_method(temp = 'linear', .... slurry_mass = 'constant') ? What do you think?

**Comment by sashahafner** [comment]
That makes some sense.
A few issues though.
First, 'constant' is the only option that seems to make sense to me.
But that is probably OK--we/users would probably have some cases where interpolation is appropriate too.
There may be a bigger complication, that this could mean a completely different approach to the fill/empty simulations.
So far we have used the times given in input for our intervals.
For filling (addition), we effectively do linear interpolation, by getting a loading rate from mass data.
Emptying is instant, as you mentioned (this could be separated from the combination of consecutive emptying events, discussed in #30 recently).
If we added the option to change these approaches it would mean some significant code changes.
In contrast, for temperature, pH, ventilation (others?) those variable values are just used in rates calculations, and how interpolation is done doesn't affect the structure of our simulation approach.

So, not impossible, but some effort and I am not sure it is important to provide this flexibility.

One alternative (in addition to removing the combination of consecutive emptying events) is to just adjust emptying time to the middle--one of your suggestions above.
I mean offer that as an option.
That could be done pretty simply within the current simulation structure, I think.
